front/app/services/participationContexts.ts

```diff
@@ -69,8 +69,8 @@ export function getInputTerm(
     // (2020/12/9): When a new timeline project is created, phases will initially
     // be []. To make sure we don't break copy that depends on an input_term,
     // we have the fallback to idea here in that case.
-    if (!isNilOrError(phaseFromUrl)) {
-      return getPhaseInputTerm([phaseFromUrl]);
+    if (!isNilOrError(phase)) {
+      return getPhaseInputTerm([phase]);
     } else if (!isNilOrError(phases) && phases.length > 0) {
```

Comment:
Renamed the prop to just "phase" since it's not always a phase from a URL param being passed in.